nil-docs/zkllvm/getting-started/installation.mdx

```diff
@@ -0,0 +1,259 @@
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+# Installation and configuration
+
+## Installing dependencies
+
+Before installing zkLLVM, install the following dependencies.
+
+- [**Boost**](https://www.boost.org/) >= 1.76.0
+- [**CMake**](https://cmake.org/) >= 3.5
+- [**Clang**](https://clang.llvm.org/) >= 12.0
+- [**Python**](https://www.python.org/) >= 3.7
+- [**Rust**](https://www.rust-lang.org/) >= 1.68 (only if there is a need to use the Rust frontend)
+
+Every dependency apart from Rust can be installed by executing the following command.
+
+```bash
+sudo apt install build-essential libssl-dev cmake clang-12 git curl pkg-config zlib
+```
+
+To install Rust, execute the following command.
+
+```bash
+curl --proto '=https' --tlsv1.2 https://sh.rustup.rs -sSf | sh
+```
+
+:::tip[Ninja]
+
+The [**Ninja build system**](https://ninja-build.org/) can make building zkLLVM from sources faster and more convenient. While it is not a direct dependency, using Ninja is preferable to simply using Unix makefiles. Install Ninja by running the below command.
+
+```bash
+sudo apt-get install ninja-build
+```
+
+:::
+
+:::tip[Installing Boost]
+
+Installing Boost from source is the most consistent way to ensure that the latest version of Boost is installed that matches the above version constraint.
+
+To install Boost from source, do the following.
+
+1. Navigate to the folder where you want to install Boost and download Boost.
+
+```bash
+wget https://boostorg.jfrog.io/artifactory/main/release/1.84.0/source/boost_1_84_0_rc1.tar.gz
+```
+
+2. Extract the archive with Boost.
+
+```bash
+cd boost_1_84_0
+```
+
+3. Prepare the build system for Boost by running `bootstrap.sh`.
+
+```bash
+./bootstrap.sh
+```
+
+4. Install Boost system-wide.
+
+```bash
+sudo ./b2 install
+```
+
+:::
+
+## Installing zkLLVM as a packagage
+
+zkLLVM is distributed as a Deb package which can be installed using the following commands.
+
+```bash
+echo 'deb [trusted=yes]  http://deb.nil.foundation/ubuntu/ all main' >>/etc/apt/sources.list
+apt update
+apt install -y zkllvm
+```
+
+This step will add `zkllvm` to the list of installed packages but it will not build all of the required components including `assigner`. This can only be achieved by performing a full installation from sources.
```

Comment:
Wait, `assigner` is packed into deb-package.

zkllvm deb-package is ready-to-use package. It contains following binaries:

- `assigner`
- `clang`
- `llvm-link`
- `transpiler`

Reply:
Thank you, I have removed this inaccuracy